server/src/main/java/org/opensearch/indices/SystemIndexDescriptor.java

```diff
@@ -33,15 +33,17 @@
 package org.opensearch.indices;
 
 import org.apache.lucene.util.automaton.CharacterRunAutomaton;
+import org.opensearch.common.annotation.PublicApi;
 import org.opensearch.common.regex.Regex;
 
 import java.util.Objects;
 
 /**
  * Describes a system index. Provides the information required to create and maintain the system index.
  *
- * @opensearch.internal
+ * @opensearch.api
  */
+@PublicApi(since = "2.16.0")
```

Comment:
@reta Why did this not cause a problem because it is returned from a public method in [SystemIndexPlugin](https://github.com/opensearch-project/OpenSearch/blob/14704b30cafadcfdcd7cd81961ee07452804b0c3/server/src/main/java/org/opensearch/plugins/SystemIndexPlugin.java#L55)?

Reply:
> @reta Why did this not cause a problem because it is returned from a public method in [SystemIndexPlugin](https://github.com/opensearch-project/OpenSearch/blob/14704b30cafadcfdcd7cd81961ee07452804b0c3/server/src/main/java/org/opensearch/plugins/SystemIndexPlugin.java#L55)?

@andrross we have not covered all APIs yet sadly :( I have finished `Plugin` & all that stems from it, `SystemIndexPlugin` & others are not annotated yet

Reply:
Got it, thanks!